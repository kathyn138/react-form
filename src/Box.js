import React, { Component } from 'react';

class Box extends Component {
  render() {
    return (
      <div>
        <button onClick={this.props.delete}>X</button>
        <br></br><div style={{ height: `${this.props.height}px`, width: `${this.props.width}px`, backgroundColor: this.props.backgroundColor }}></div>
      </div>
      )
  }
}

export default Box;