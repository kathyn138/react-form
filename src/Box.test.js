import React from 'react';
import Box from './Box';
import { shallow } from 'enzyme';
import toJson from "enzyme-to-json";


it('renders without crashing', () => {
  shallow(<Box />)
});

it('matches snapshot', function() {
  let wrapper = shallow(<Box />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
})
