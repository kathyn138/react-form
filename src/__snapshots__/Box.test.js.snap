// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <button>
    X
  </button>
  <br />
  <div
    style={
      Object {
        "backgroundColor": undefined,
        "height": "undefinedpx",
        "width": "undefinedpx",
      }
    }
  />
</div>
`;
